api/src/Page/Shipment.php

```diff
@@ -925,6 +1000,27 @@ function _dispatch_dewar() {
                 array($dew['DEWARID'], $dewar_location)
             );
 
+            $data = $this->args;
+
+            $terms = $this->db->pq(
+                "SELECT cta.couriertermsacceptedid FROM couriertermsaccepted cta WHERE cta.shippingid=:1",
+                array($dew['SHIPPINGID'])
+            );
+            $terms_accepted = sizeof($terms) ? true : false;
+
+            if (isset($use_shipping_service) && $use_shipping_service && $terms_accepted && in_array($country, $facility_courier_countries)) {
```

Comment:
isset($use_shipping_service) && $use_shipping_service

Is this a common pattern maybe put it in a utility class as a function:
   getValue(value, default)  which returns a default value if not set